static/docs/user-guide/contributing.md

```diff
@@ -281,9 +281,13 @@ To run DVC from its Git repository you need to setup your environment:
 
 ## Code style guidelines
 
-We are using [PEP8](https://www.python.org/dev/peps/pep-0008/?) and are checking
+We are using [PEP8](https://www.python.org/dev/peps/pep-0008/?) and checking
 that our code is formatted with [black](https://github.com/ambv/black).
 
+For [docstrings](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring),
```

Comment:
probably not only docstrings? cc @efiop

Reply:
@mroutis mentioned the docstrings to me recently but yes maybe more aspects? Please lmk and I'll update the doc. Thanks

Reply:
@shcheklein We don't follow google coding style in full, so It is hard to recommend it here, since black will probably reformat it anyway. Let's start with this docstring reference, we do use those.

Reply:
👍